### DISCUSSION TITLE
Customisable Disambiguation Strings when Importing

### DISCUSSION THREAD

**Discussion by Serene-Arc** [beetbox/beets#4545] [Ideas]
When importing audiobooks through the beets-audible plugin, there is often an issue with multiple audiobooks being difficult to distinguish since the composer/narrator is a core quality and it isn't displayed. A simple fix that isn't limited to just the audible plugin would be allowing customisation of the debug string that is displayed when importing objects. Obviously it is fine for most imports but there are occasions where it could be useful. Leaving the defaults as the current configuration would also ensure minimal disruption.

**Comment by sampsyo** [comment]
Great idea. I don't see a downside in making this configurable—hopefully it's not too hard to replace that hard-coding with a configurable list of fields!

**Reply by Serene-Arc** [reply]
I have written an example for my own use and there are still bugs, but it works in general. I'll open a PR!
